_posts/03-02-01-Programming-Paradigms.md

```diff
@@ -3,51 +3,51 @@ isChild: true
 anchor:  programming_paradigms
 ---
 
-## Programming Paradigms {#programming_paradigms_title}
+## প্রোগ্রামিং এর ধরন {#programming_paradigms_title}
 
-PHP is a flexible, dynamic language that supports a variety of programming techniques. It has evolved dramatically over
-the years, notably adding a solid object-oriented model in PHP 5.0 (2004), anonymous functions and namespaces in
-PHP 5.3 (2009), and traits in PHP 5.4 (2012).
+পিএইচপি একটি পরিবর্তনশীল ও শক্তিশালী ল্যাঙ্গুয়েজ। এটি বিভিন্ন ধরনের প্রোগ্রামিং টেকনিক সমর্থন করে। গত কয়েক বছরে এর ইতিবাচক পরিবর্তন লক্ষ্য করা যায়,
+বিশেষ করে পিএইচপি ৫.০ (২০০৪) ভার্সন এ নির্ভরযোগ্য অবজেক্ট-ওরিয়েন্টেড মডেল এর সংযোজন, পিএইচপি ৫.৩ (২০০৯) ভার্সন এ অ্যাননিমাস ফাংশন এবং নেইমস্পেস এর সংযোজন,
+এবং পিএইচপি ৫.৪ (২০১২) ভার্সন এ ট্রেইটস এর সংযোজন।
 
-### Object-oriented Programming
+### অবজেক্ট-ওরিয়েন্টেড প্রোগ্রামিং
 
-PHP has a very complete set of object-oriented programming features including support for classes, abstract classes,
-interfaces, inheritance, constructors, cloning, exceptions, and more.
+পিএইচপি তে পরিপূর্ণভাবে অবজেক্ট-ওরিয়েন্টেড প্রোগ্রামিং এর বৈশিষ্ট্যগুলো বিদ্যমান, যেমন ক্লাস, অ্যাবস্ট্রাক্ট ক্লাস,
+ইন্টারফেইস, ইনহেরিটেন্স, কনস্ট্রাক্টর, ক্লোনিং, এক্সেপশন, ইত্যাদি।
```

Comment:
The last element of a list should not have separator comma:
"এক্সেপশন, ইত্যাদি" => "এক্সেপশন ইত্যাদি"

Reply:
OK vai